components/region_cache_memory_engine/src/keys.rs

```diff
@@ -180,9 +180,9 @@ pub fn decode_key(encoded_key: &[u8]) -> InternalKey<'_> {
 }
 
 /// Format for an internal key (used by the skip list.)
-/// ```
-/// contents:      key of size n     | value type | sequence number shifted by 8 bits
-/// byte position:         0 ..  n-1 | n          |  n + 1 .. n + 7
+///
+/// ```text
+/// Format: | user key (n bytes) | value type (1 bytes) | sequence number (7 bytes) |
```

Comment:
According to https://github.com/facebook/rocksdb/blob/v9.2.1/db/dbformat.h#L178 , the format should be 
`user key | seq number (7 bytes) | value type (1 byte)`

Reply:
emm, seems after encode by little endian, the format is `user key | value type (1 byte) | seq number (7 bytes)`.

Reply:
It's encoded in little endian, so in the bytes representation, "value type" should be ahead of "seq number".